### PR TITLE
fix qavan entry in German + other

### PR DESCRIPTION
**Pull request by LievenLitaer**
The user comment was right about "ich grüße dich/dir". But I decided to fix it differently, because "grüße" is "greeting", but not "salute". So I changed that to "salutieren" which is then correct using "dir" (it's possible that's the reson it was wrong in the first place)

### CODE REVIEW

File: KlingonAssistant/data/mem-12-q.xml [hunk 1]
```diff
@@ -1243,12 +1243,12 @@ The subtitle was "What's she doing?" but a more literal translation is "Why is s
...
       <column name="part_of_speech">sen:bc</column>
       <column name="definition">I salute you!</column>
-      <column name="definition_de">Ich grüße dir!</column>
+      <column name="definition_de">Ich salutiere dir!</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also"></column>
-      <column name="notes">This is used when addressing a single person. To salute multiple people, say {Savan!:sen}</column>
-      <column name="notes_de"></column>
+      <column name="notes">This is used when addressing a single person. To salute multiple people, say {Savan!:sen}.</column>
```
Comment (De7vID) [inline KlingonAssistant/data/mem-12-q.xml:1250]: There shouldn't be a period. The sentence already ends in an exclamation mark.

File: KlingonAssistant/data/mem-12-q.xml [hunk 2]
```diff
@@ -1303,14 +1305,14 @@ The subtitle was "What's she doing?" but a more literal translation is "Why is s
...
-      <column name="definition_de"></column>
+      <column name="definition_de">Gedächtnis, Erinnerungsvermögen</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{qaw:v}</column>
       <column name="notes">This refers to "memory" in the sense of the ability to remember or the power of recall. It does not refer to specific memories or recollections, for which see {wov'on:n}.
 
 Someone who has an excellent memory may be said to have a {qaw pov:n@@qaw:n, pov:v}.</column>
-      <column name="notes_de"></column>
+      <column name="notes_de">Wenn jemand ein gutes Gedächtnis hat, kann man sagen, er habe ein {qaw pov:n@@qaw:n, pov:v}</column>
```
Comment (De7vID) [inline KlingonAssistant/data/mem-12-q.xml:1315]: This is only the last sentence. Did you mean to leave the others out?

File: KlingonAssistant/data/mem-12-q.xml (hunk 1 above)
Comment (LievenLitaer) [reply KlingonAssistant/data/mem-12-q.xml:1250]: Okay. Thanks fo telling me. I thought the exclamation mark was part of the definition.

File: KlingonAssistant/data/mem-12-q.xml (hunk 2 above)
Comment (LievenLitaer) [reply KlingonAssistant/data/mem-12-q.xml:1315]: Yes, indeed. The German definition is not ambiguous as the english word, so there is no need to distinguish.

**Pull request merged**